iosApp/iosApp/Pages/StopDetails/StopDetailsUnfilteredView.swift

```diff
@@ -90,31 +94,37 @@ struct StopDetailsUnfilteredView: View {
-                                            textColor: Color.text,
-                                            onViewDetails: {
-                                                nearbyVM.pushNavEntry(.alertDetails(
-                                                    alertId: alert.id,
-                                                    line: nil,
-                                                    routes: nil,
-                                                    stop: stop
-                                                ))
-                                                analytics.tappedAlertDetails(
-                                                    routeId: "",
-                                                    stopId: stopId,
-                                                    alertId: alert.id,
-                                                    elevator: true
-                                                )
-                                            }
-                                        )
+                                if stopDetailsVM.showElevatorAccessibility, hasAccessibilityWarning {
+                                    if !departures.elevatorAlerts.isEmpty {
+                                        ForEach(departures.elevatorAlerts, id: \.id) { alert in
+                                            AlertCard(
+                                                alert: alert,
+                                                spec: .elevator,
+                                                color: Color.clear,
+                                                textColor: Color.text,
+                                                onViewDetails: {
+                                                    nearbyVM.pushNavEntry(.alertDetails(
+                                                        alertId: alert.id,
+                                                        line: nil,
+                                                        routes: nil,
+                                                        stop: stop
+                                                    ))
+                                                    analytics.tappedAlertDetails(
+                                                        routeId: "",
+                                                        stopId: stopId,
+                                                        alertId: alert.id,
+                                                        elevator: true
+                                                    )
+                                                }
+                                            )
+                                            .padding(.horizontal, 16)
+                                            .padding(.bottom, 16)
+                                        }
+                                    }
+                                } else {
+                                    NotAccessibleCard()
```

Comment:
issue: I don't think the NotAccessibleCard should be showing up at Government Center - it correctly gets the ISA icon in nearby transit. On android there is no NotAccessibleCard as expected, maybe some small difference in the logic there! 

thought(non-blocking): - this makes me think if there are ways we could approach android implementation differently in the future to share more logic with iOS. Would a common function given accessibility inputs that returns an enum value depending on the conditional branch case you're in have helped? Would love to know if you have any ideas about that from your experience porting to iOS!

Reply:
Yeah, I'm with you on being nervous about the feasibility of fully shared VMs! It is a good callout though - if more of the logic is in VMs, I think it will be easier to recognize & extract utils that could be common to the VMs, even if the representation of state / lifecycle objects stay native.